ngraph/test/frontend/paddlepaddle/op.cpp

```diff
@@ -171,9 +171,16 @@ namespace fuzzyOp {
                 throw std::runtime_error("not supported dtype out "+ dtype);
             }          
         }
-            
-        test_case.run_with_tolerance_as_fp();
-        // test_case.run();
+
+        if (std::all_of(results.begin(), results.end(), [](const std::shared_ptr<ngraph::op::v0::Result> &result) {
+            return result->get_element_type() == element::i32;
```

Comment:
should other 'integer' types be covered? I expected that in the loop above you'll set 'bool has_float_output = true' if 'dtype == "<f4"' and run test case depending on this variable

Reply:
Hi, we only test int32 model but not int64 model is because during inference IE converts int64 model to int32 model under some cases. When comparing the int32 inferenced result & int64 expected result, they always mismatch. Therefore I just enable the int32 model here. But we can still add int64 support here for extension.